manifests/integrations/ceph.pp

```diff
@@ -3,16 +3,24 @@
 # This class will install the necessary configuration for the ceph integration
 #
 # Parameters:
+#   $tags
+#       Optional array of tags
+#   $ceph_cmd
+#       Optional ceph cmd
 
 # Sample Usage:
 #
 #  class { 'datadog_agent::integrations::ceph' :
 #  }
 #
 class datadog_agent::integrations::ceph(
+  $tags = [ 'name:ceph_cluster' ],
```

Comment:
Ugh... I kind of hate that we have to do it this way, but it is indeed the only way to keep backward compatibility 👍